app/assets/javascripts/add_date_field.js.coffee

```diff
@@ -10,7 +10,18 @@
         validateOnBlur: false
       })
 
+    date_multiple_default = () ->
+      $('#date_wrapper .date-multiple .date-input input').datetimepicker({
+        timepicker:false,
+        format:'Y-m-d',
+        scrollMonth: false,
+        scrollTime: false,
+        scrollInput: false,
+        validateOnBlur: false
+      })
+
     date_default()
+    date_multiple_default()
```

Comment:
Pass the full broad selector to bind datetimepicker to here `date_multiple_default($('#date_wrapper .date-multiple .date-input input'))`